src/plugins/gui/RewardsCardPlugin.tsx

```diff
@@ -0,0 +1,132 @@
+import { EdgeSpendInfo } from 'edge-core-js'
```

Comment:
It's because it exports `makeRewardsCardPlugin` which is to say returns the RewardsCardPlugin "type". But I don't really care either way so I'll rename it if you think it's clearer.